Cargo.toml

```diff
@@ -46,21 +46,25 @@ rustls-pemfile = "2.0.0"
 rustls-pki-types = "1.1.0"
 secrecy = { version = "0.8.0", features = ["alloc", "serde"] }
 serde = "1.0.154"
-serde_json = "1.0.94"
+serde_json = "1.0.117"
 tracing = "0.1"
-zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main", features = [
+zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", features = [
   "plugins",
   "unstable",
+  "internal",
 ] }
-zenoh-collections = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-core = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-ext = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main", features = [
+zenoh-collections = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0" }
```

Comment:
internal crates are not needed now, only `zenoh` is enough.
For plugins `zenoh-plugin-trait` is necessary also because of macro exports from it and that's all

Reply:
Perfect. https://github.com/eclipse-zenoh/zenoh-plugin-mqtt/pull/131 first then we can merge this branch as an update after the fact.